synapse_data_warehouse/synapse/dynamic_tables/V2.21.0__verificationsubmission_latest.sql

```diff
@@ -0,0 +1,24 @@
+use schema {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP01
+
+CREATE DYNAMIC TABLE IF NOT EXISTS VERIFICATIONSUBMISSION_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+    -- We deduplicate simply by selecting the latest record for each
+    -- verification submission ID...
+    WITH latest_unique_rows AS (
+        SELECT
+            verificationsubmissionsnapshots.*,
+        FROM
+            {{database_name}}.synapse_raw.verificationsubmissionsnapshots --noqa: TMP
+        QUALIFY ROW_NUMBER() OVER (
+                PARTITION BY id
+                ORDER BY change_timestamp DESC, snapshot_timestamp DESC
+            ) = 1
```

Comment:
Can we add `snapshot_timestamp` within last 2 weeks?

Reply:
The latest snapshot taken for `verificationsubmissionsnapshots` was Jan 11, 2024 so this would create an empty table. Is there a planned schedule for more frequent updates to this snapshots table? And maybe we should also revisit how we determine the "latest" window for these tables (i.e. why is 2 weeks from today the standard that we usually set for these). I can make a ticket to discuss this later since we're already discussing the R vs V scripts here.

cc @philerooski

Reply:
The biggest issue is it's hard to test some of this stuff on the dev database because data doesn't flow in daily.

Reply:
Hmm okay, that's good to know. This is a separate convo more related to the CI pipeline work, so I left a comment [there](https://github.com/Sage-Bionetworks/snowflake/pull/78#issuecomment-2477015135) for us to address when we get back to that

Reply:
If we do leave out the 2 week window, none of these latest tables are usable (imo) until we get a better understanding of the data within these tables (which require discussion with Platform)

Reply:
> Do you have access to the prod snapshots?

Nope, you're right, I was looking at `dev` 😅 The latest snapshot timestamp for this table on `prod` is from Nov 14, 2024, so yes I can make this query for a 2-week window